docs/src/main/sphinx/release/release-416.md

```diff
@@ -0,0 +1,12 @@
+# Release 416 (3 May 2023)
+
+## General
+
+* Improve performance of partitioned `INSERT`, `CREATE TABLE AS .. SELECT`, and
+  `EXECUTE` statements when the source table statistics are missing or 
+  inaccurate. ({issue}`16802`)
+* Improve performance of `LIKE` expressions that contain `%`. ({issue}`16167`)
```

Comment:
should we say "use pattern matching with %" instead ?

Reply:
That sounds more convoluted? What's confusing about the current phrasing?

Alternatively: "Improve performance of LIKE patterns that contain %" ?

Reply:
I like that last one best ..